src/Identity/ApiAuthorization.IdentityServer/src/IdentityServerBuilderConfigurationExtensions.cs

```diff
@@ -93,6 +93,7 @@ public static class IdentityServerBuilderConfigurationExtensions
             IConfiguration configuration)
         {
             builder.ConfigureReplacedServices();
+            builder.AddApiScopes();
```

Comment:
Scopes where separated from resources, in our model we automatically add the scopes defined on resources as ApiScopes